postman/Space Up.postman_collection.json

```
@@ -3706,6 +3706,11 @@
       "key": "gateway_base_url",
       "value": "http://localhost:4122"
     },
+    {
+      "id": "03da5b38-e050-44cc-b00a-cef12e3a7812",
+      "key": "deployed_service_idl",
```

Should be this right?
```suggestion
      "key": "deployed_service_id",
```